app/views.py

```diff
@@ -0,0 +1,339 @@
+from app import app, db
+from app.models import User, BucketList, ListItem
+from flask_jwt import JWT, jwt_required, current_identity
+from werkzeug.exceptions import HTTPException, default_exceptions
+from flask import request, g, jsonify, abort
+from app.validate import validate_register, validate_bucketlist, validate_item\
+    , validate_limit_and_offset
+
+
+def verify_password(email, password):
+    g.user = User.query.filter(email == User.email).first()
+    if g.user is not None and g.user.check_password(password):
+        return g.user
+    return False
+
+
+def identity(payload):
+    user_id = payload['identity']
+    return User.query.filter(user_id == User.user_id).first()
+
+
+jwt = JWT(app, verify_password, identity)
+
+
+def user_exists(email):
+    exists = db.session.query(db.session.query(User)
+                              .filter_by(email=email)
+                              .exists()).scalar()
+    return exists
+
+
+@app.route("/resource", methods=['POST'])
+@jwt_required()
+def get_resource():
+    return jsonify(dict(msg="Hello world"))
```

Comment:
ensure you always define the status otherwise the default status 200 will always be sent which is not always true.